### DISCUSSION TITLE
Resize picture before post

### DISCUSSION THREAD

**Discussion by sydesb** [Waboodoo/HTTP-Shortcuts#347] [Ideas]
Hi,

Photos are getting bigger and bigger on the phone. It would be nice to be able to resize the image before uploading them.

Than you for this great tool !!

**Comment by Waboodoo** [comment]
Hello, and yes, I agree with this. What do you imagine this feature would look like for your usecase? Would it be sufficient to add a file size limit and/or bounds on the image's width and height, which would automatically scale down an image if it goes above these limits? Or would it be percentage based?
